src/app/utils/item.js

```diff
@@ -227,14 +238,19 @@ function LibraryItem() {
   };
 
   /**
-   * getHoldingLocation(item)
+   * getHoldingLocation(item, nonNyplRecap)
    * Returns updated location data from the holdingLocation property in the API for each item.
    * @param {object} item
+   * @param {boolean} nonNyplRecap
    * @return {object}
    */
-  this.getHoldingLocation = (item) => {
+  this.getHoldingLocation = (item, nonNyplRecap) => {
     let location = this.getDefaultLocation();
 
+    if (nonNyplRecap) {
```

Comment:
What if we don't get any location for NYPL item? Will it still be SASB is this case and only non NYPL to be offsite?

I feel the logic probably should be,
1) offsite default, but if nypl => SASB (if this is the default for NYPL)
2) then update the default value to the returned value of the API